bundles/specmate-cause-effect-patterns/src/com/specmate/cause_effect_patterns/parse/matcher/SubtreeMatcher.java

```diff
@@ -1,7 +1,10 @@
-package com.specmate.nlp.dependency.matcher;
+package com.specmate.cause_effect_patterns.parse.matcher;
 
 import com.google.common.base.Optional;
```

Comment:
please use java.util.Optional, if there is no specific reason you use the guava lib